packages/config/src/layer2s/metis.ts

```diff
@@ -239,13 +245,20 @@ export const metis: Layer2 = {
   ],
   contracts: {
     addresses: [
-      discovery.getContractDetails(
+      // note: these three contracts are not used anymore - transaction batch hashes are posted to EOA. Note that these contracts are still being discovered
+      /*discovery.getContractDetails(
         'MVM_CanonicalTransaction',
         'MVM CanonicalTransaction is a wrapper of Canonical Transaction Chain that implements optimistic data availability scheme L1. If Sequencer is not malicious, it simply forwards appendSequencerBatch() calls to CanonicalTransactionChain.',
       ),
+      */
+      discovery.getContractDetails(
+        'MVM_Verifier',
+        'This contract implements a voting scheme with which the majority of Verifiers can challenge malicious state roots proposed. There are no whitelisted verifiers, hence this contract is not used in practice.',
+      ),
+
       discovery.getContractDetails(
         'CanonicalTransactionChain',
-        'The Canonical Transaction Chain (CTC) contract is an append-only log of transactions which must be applied to the OVM state. It defines the ordering of transactions by writing them to the CTC:batches instance of the Chain Storage Container. CTC batches can only be submitted by OVM_Sequencer. The CTC also allows any account to enqueue() a transaction, which the Sequencer must eventually append to the rollup state.',
+        'The Canonical Transaction Chain (CTC) contract is an append-only log of transactions which must be applied to the OVM state. Given that transactions batch hashes are sent to an EOA address, it allows any account to enqueue() a transaction, which the Sequencer must eventually append to the rollup state.',
       ),
       discovery.getContractDetails(
         'StateCommitmentChain',
```

Comment:
We should edit this description similarly to the CTC one, because 1:1 correspondence with CTC does not get checked anymore [(see commented-out part in the SC)](https://etherscan.io/address/0x6b1cd90796dc97c859734dda4ff7816f703ddb48#code#F1#L244).

Reply:
(should i do it?)